srcpkgs/librespeed-cli/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'librespeed-cli'
+pkgname=librespeed-cli
+version=1.0.10
+revision=1
+build_style=go
+go_import_path=github.com/librespeed/speedtest-cli
+short_desc="LibreSpeed command line tool"
+maintainer="Massimo Manzato <massimo.manzato@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/librespeed/speedtest-cli"
+distfiles="https://github.com/librespeed/speedtest-cli/archive/refs/tags/v${version}.tar.gz"
+checksum=904ec84f41714d5a7ab778534d332219cd254fdd0f97cc33ebb3540d31fb802c
+
+post_build() { # Rename the executable due to conflic with PKG speedtest-cli
+	find  "${GOPATH}/bin/" -name "speedtest-cli" -print0 | while IFS= read -d '' F ; do mv "$F" "${F/%speedtest-cli/librespeed-cli}" ; done
+}
```

Comment:
```suggestion
post_install() {
	mv "${DESTDIR}"/usr/bin/{speedtest,librespeed}-cli
}
```

Reply:
Hi [classabbyamp](https://github.com/classabbyamp)

.... this fail with this error: 

`mkdir -p $WORK/b001/exe/
cd .
/usr/lib/go/pkg/tool/linux_amd64/link -o $WORK/b001/exe/a.out -importcfg $WORK/b001/importcfg.link -buildmode=exe -buildid=UGLpSgEbrhVq2b7jm8SN/adCN0K-eHLQVQR0HCyM-/LnvtcmFsGxEd6JXJ-Pov/UGLpSgEbrhVq2b7jm8SN -extld=cc /host/gocache-muslc/d3/d37a06acba36938d11acc6a9b2cf8f92b1eb99b56fb63f3b7d7374ffe9d84478-d
/usr/lib/go/pkg/tool/linux_amd64/buildid -w $WORK/b001/exe/a.out # internal
mkdir -p /builddir/librespeed-cli-1.0.10/_build-librespeed-cli-xbps/bin/
mv $WORK/b001/exe/a.out /builddir/librespeed-cli-1.0.10/_build-librespeed-cli-xbps/bin/speedtest-cli
rm -r $WORK/b001/
=> librespeed-cli-1.0.10_1: running post_build ...
mv: cannot stat '/destdir//librespeed-cli-1.0.10/usr/bin/speedtest-cli': No such file or directory
=> ERROR: librespeed-cli-1.0.10_1: post_build: 'mv "${DESTDIR}"/usr/bin/{speedtest,librespeed}-cli' exited with 1
=> ERROR:   in post_build() at srcpkgs/librespeed-cli/template:15
`

Reply:
it should be post_install

Reply:
Hi @classabbyamp
You are right !! 👍
I apologize, I haven't read your suggestion with attention! 
Max